README.md

```diff
@@ -136,9 +136,10 @@ instance, or the system browser.
 
     iOS supports these additional options:
 
+    - __usewkwebview__: set to `yes` to use WKWebView engine for the InappBrowser. Omit or set to `no` (default) to use UIWebView. Note: this requires that a WKWebView engine plugin be installed in the Cordova project (e.g. [cordova-plugin-wkwebview-engine](https://github.com/apache/cordova-plugin-wkwebview-engine) or [cordova-plugin-ionic-webview](https://github.com/ionic-team/cordova-plugin-ionic-webview)).
```

Comment:
What will happen if no WKWebView plugin is installed?

Reply:
If `usewkwebview=yes` and no WKWebView plugin is installed, [the `loaderror` callback will be invoked](https://github.com/dpa99c/cordova-plugin-inappbrowser/blob/CB-7179/src/ios/CDVInAppBrowser.m#L48) with the error message "usewkwebview option specified but cordova-plugin-wkwebview-engine not present"

Reply:
OK, I've made tweaks based on those suggestions